content/courses/testing.md

```diff
@@ -6,19 +6,22 @@ slug: "/courses/Testing/"
 category: "Automated Testing"
 tags:
   - "Jest"
-description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into 
-frontend testing using Jest and eventually Enzyme. By the end of this track you will know how 
-to test user interfaces, as well as user interactions." 
-lessons: 
+description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into
+frontend testing using Jest, and eventually Enzyme or React Testing Library. By the end of this track you will know how
+to test user interfaces, as well as user interactions."
+lessons:
   - link: "intro-to-unit-testing/"
     title: Intro to Unit Testing
     description: "Learn the basics of unit tests in Javascript. This session will show you how to
     write assertions, help you test basic JS functions, and show you why tests can be so helpful. By the end of this session you
     will be able to write a unit test for a vanilla JS function."
-  - link: "component-testing/"
-    title: Component Testing
+  - link: "component-testing-enzyme/"
+    title: Component Testing with Enzyme
     description: "This session will show you the basics of testing User Interfaces. You will start with Jest only, learning
     the difference between React Components and Elements. From there, you will add in Enzyme to test component instances and their many variations. By the end of this session you will be able to write a Jest/Enzyme test for a React component."
+  - link: "component-testing-rtl/"
+    title: Component Testing with React Testing Library
+    description: "This session covers component testing with React Testing Library, an alternative to Enzyme with a different philosophy and API. By the end of this session you will be able to write a test for a React component using Jest/React Testing Library."
```

Comment:
We're using both Jest and RTL, can you remove the slash on this line and replace it with an "and"?